html/recordplaytest.js

```diff
@@ -147,7 +147,7 @@ $(document).ready(function() {
 												recordplay.createAnswer(
 													{
 														jsep: jsep,
-														media: { audioSend: false, videoSend: false },	// We want recvonly audio/video
+														media: { audioSend: false, videoSend: false, data: true },	// We want recvonly audio/video
```

Comment:
This has no effect on the demo - it will still work if there are no data channels on the recoring. On the other hand, if not present and someone wants to test using recordings from the streaming plugin, it would need a change to work - datachannel will not open from server side otherwise.

Reply:
You're right, sorry! I thought this was the recording part, not the playout one :see_no_evil: